apps/server/src/modules/authentication/strategy/ws-jwt.strategy.spec.ts

```diff
@@ -53,39 +43,49 @@ describe('jwt strategy', () => {
 
 	describe('when authenticate a user with jwt', () => {
 		const setup = () => {
+			const mockJwtPayload = jwtPayloadFactory.build();
```

Comment:
to avoid code duplication maybe we can define the const mockJwtPayload = jwtPayloadFactory.build(); for the whole class and use it everywhere?

Reply:
the review comment is to add factory and move this to setup methods

Reply:
https://github.com/hpi-schul-cloud/schulcloud-server/compare/main...EW-601-review-comments#diff-80d713292e85f709b067d70a7a84582f72dfc73fd2a145b14ddb9fa10e2af4bd